src/config/config-helper.js

```diff
@@ -117,6 +122,16 @@ export function buildSearchOptionsFromConfig() {
     };
   }
 
+  if (config.thumbnailField) {
+    resultFields[config.thumbnailField] = {
+      raw: {},
+      snippet: {
+        size: 100,
+        fallback: true
```

Comment:
A little confused about the snippet part of this, what purpose does it serve here? Wouldn't we want raw so that the img URL doesn't get accidentally truncated if it's super long? 🤔

Reply:
Yeah that's a good point. We request to receive snippet and raw values. We don't *need* the snippet value, but we MUST have the raw value, as it's used by the `Results` component.

So this check is really just to ensure that we have the `raw` value available.

And while we don't *need* the snippet value, we're adding it here just in case a user needs it in their UI. I suppose we're favoring having it and not needing it over a user needing and not having it available.

Reply:
Cool, no worries! Thanks for the clarification/context, super appreciated for my own knawledge!